### ISSUE TITLE
Uncomment a block with empty lines does not work with `ignore_blank_lines = true`

### ISSUE THREAD

**Issue by UtkarshVerma** [echasnovski/mini.nvim#289]
### Contributing guidelines

- [X] I have read [CONTRIBUTING.md](https://github.com/echasnovski/mini.nvim/blob/main/CONTRIBUTING.md)
- [X] I have read [CODE_OF_CONDUCT.md](https://github.com/echasnovski/mini.nvim/blob/main/CODE_OF_CONDUCT.md)
- [X] I have updated 'mini.nvim' to latest version

### Module(s)

mini.comment

### Description

I think a video would be better. In the video, I wish to comment and uncomment a block, but I can't uncomment it.

https://user-images.githubusercontent.com/31820255/229094109-c6335647-97c0-44aa-93d7-090a64723602.mp4

### Neovim version

0.8.3

### Steps to reproduce

Set `ignore_blank_lines = true` in lua config for `mini.comment`.

### Expected behavior

Basically, with `ignore_blank_lines` option set to true, I would expect empty lines to be ignored during the uncommenting part as well.

### Actual behavior

`mini.comment` is not able uncomment blocks if a blank line is there inside the block.

**Comment by echasnovski** [comment]
Thanks for the issue!

I agree, it does seem unnatural when `options.ignore_empty_line` is `true`. I'll see what I can do here.

**Comment by UtkarshVerma** [comment]
Works as expected. Thanks a lot!
